media/stylus/components/wiki/spec.styl

```diff
@@ -0,0 +1,108 @@
+/*
+Colors and Badges for the Spec2 macro template
+====================================================================== */
+
+.spec-ED {
+    color: #e61919;
+}
+
+.spec-WD,
+.spec-Draft {
+    color: #e66e33;
+}
+
+.spec-CR,
+.spec-RC {
+    color: #00978e;
+}
+
+.spec-PR {
+    color: #72ac35;
+}
+
+.spec-REC,
+.spec-RFC {
+    color: #309c40;
+}
+
+.spec-Living,
+.spec-Standard {
+    color: #00539f;
+}
+
+/*
+Badges
+-------------------------------------------------------------- */
+.spec-ED:before,
+.spec-WD:before,
+.spec-CR:before,
+.spec-RC:before,
+.spec-PR:before,
+.spec-REC:before,
+.spec-RFC:before,
+.spec-Living:before,
+.spec-Standard:before,
+.spec-Draft:before {
+    font: 0.7em Helvetica, Arial, sans-serif;
+    font-weight: 700;
+    border-radius: 2px;
+    border-left: 4px solid;
+    letter-spacing: 1px;
+    padding: 0.45em 0.35em 0.30em;
+    /* bidi for this is complicated, the tag itself is LTR but the
+       name of the spec could be either LTR or RTL (depending on
+       if it's been translated) so we add margins to both sides.
+       Not perfect but good compromise.  */
+    bidi-value(margin, 0.2em 0.6em 0 0, 0.2em 0.6em 0);
```

Comment:
The content inside the tag is reliably LTR. The content outside the tag is not reliably RTL or LTR and so the tag is unpredictably placed on either the left or the right of the content it sits before. On RTL pages it needs margins on both side but on LTR pages it is predictably placed on the left and does not need the left margin.